### PR TITLE
Added span tag to ExamsView options text(fixes #5203)

### LINKED ISSUE #5203: ExamsView: Choice text overflow
**Current Behavior**

In a multiple choice question if the text for a choice is too long it will overflow off the screen

**Desired Behavior**

If the choice for a question is too long it should wrap to another line

image

### PR DESCRIPTION
**Pull request by sjkadali**
Fixes #5203

### CODE REVIEW

**Comment by sjkadali** [comment]
@paulbert , @lmmrssa please review the PR. Thanks!

**Comment by sjkadali** [comment]
@paulbert have made the requested changes. Please review, thanks!

File: src/app/exams/exams-view.scss [hunk 1]
```diff
@@ -5,11 +5,12 @@
 
     mat-radio-button, mat-checkbox {
       margin: 5px;
+      float: left;
     }
 
     mat-radio-button:last-child, mat-checkbox:last-child {
       margin-bottom: 15px;
-    }
+    }
```
Comment (paulbert) [inline src/app/exams/exams-view.scss]: Please be careful to not add extra spaces in your PRs.

File: src/app/exams/exams-view.component.html [hunk 2]
```diff
@@ -28,9 +28,10 @@
             <planet-markdown-textbox [formControl]="answer"></planet-markdown-textbox>
           </mat-form-field>
           <mat-radio-group *ngSwitchCase="'select'" class="question-list" [formControl]="answer">
-            <mat-radio-button *ngFor="let option of question.choices" [value]="option">
-              {{option.text}}
-            </mat-radio-button>
+            <div *ngFor="let option of question.choices">
+              <mat-radio-button  [value]="option"></mat-radio-button>
```
Comment (paulbert) [inline src/app/exams/exams-view.component.html]: Please remove the double space here

Comment (sjkadali) [reply src/app/exams/exams-view.component.html]: @paulbert have made the requested changes. Please review the changes. Thanks!

**Pull request merged**